blockchain/process.go

```diff
@@ -80,7 +80,7 @@ func (b *BlockChain) checkKnownInvalidBlock(node *blockNode) error {
 // checkBlockHeaderPositional.
 //
 // This function MUST be called with the chain lock held (for writes).
-func (b *BlockChain) maybeAcceptBlockHeader(header *wire.BlockHeader, flags BehaviorFlags, checkHeaderSanity bool) (*blockNode, error) {
+func (b *BlockChain) maybeAcceptBlockHeader(header *wire.BlockHeader, checkHeaderSanity bool) (*blockNode, error) {
```

Comment:
The comment should be modified to remove the blurb about the `flags` param since it is being removed imo.

Reply:
Good call, updated.